docs/guides/deployment/fly_io.rst

```diff
@@ -77,16 +72,20 @@ Let's discuss what's going on with all these secrets.
   of the default 5656, because Fly.io prefers ``8080`` for its default health
   checks.
 
-Finally, let's scale the VM as EdgeDB requires a little bit more than the
-default Fly.io VM side provides:
+Finally, let's configure the VM size as EdgeDB requires a little bit more than
```

Comment:
We've reduced server memory requirements significantly in 4.x.  See if it works with smaller memory allocations.  512 should definitely be enough.

Reply:
512mb seems to work fine. I updated the example toml file.